src/client_comms.js

```diff
@@ -39,8 +39,12 @@ class ClientComms {
     this.io = new Server(
       expressServer,
       {
-        pingInterval: PING_INTERVAL_MS,
-        pingTimeout: PING_TIMEOUT_MS
+        cors: {
+          origin: RQ_PARAMS.EXTERNAL_UI_ORIGIN,
+          methods: ['GET', 'POST']
+        },
+        pingInterval: PING_INTERVAL,
```

Comment:
This branch doesn't run because this constant isn't defined. It looks to be a simple typo, from moving the lines around. You likely edited an out-of-date version of this file.